src/libs/actions/Policy/Policy.ts

```diff
@@ -3075,6 +3075,10 @@ function getAdminPoliciesConnectedToSageIntacct(): Policy[] {
     return Object.values(allPolicies ?? {}).filter<Policy>((policy): policy is Policy => !!policy && policy.role === CONST.POLICY.ROLE.ADMIN && !!policy?.connections?.intacct);
 }
 
+function getAdminPoliciesConnectedToNetSuite(): Policy[] {
+    return Object.values(allPolicies ?? {}).filter<Policy>((policy): policy is Policy => !!policy && policy.role === CONST.POLICY.ROLE.ADMIN && !!policy?.connections?.netsuite);
+}
```

Comment:
NAB: Instead of a method for each integration we could do `getAdminPoliciesConnectedToIntegration` and allow for a ConnectionName to be passed in.

Reply:
I thought so. Will try to take care of it in any future PR.